tests/source/issue_5533.rs

```diff
@@ -0,0 +1,6 @@
+// rustfmt-format_code_in_doc_comments: true
+
+struct TestStruct {
+    position_currency: String, // Currency for position of this contract. If not null, 1 contract = 1 positionCurrency.
+    pu: Option<i64>, // Previous event update sequense ("u" of previous message), -1 also means None
+}
```

Comment:
The `source` file isn't necessary since no formatting changes are expected. Only the `target` file is needed to test that the input isn't changed.